app/services/typicode.js

```diff
@@ -0,0 +1,27 @@
+const request = require('request-promise'),
+  typicodePath = 'https://jsonplaceholder.typicode.com',
+  apiError = require('../errors');
```

Comment:
I think you meant `{apiError} = require('../errors');`. As it is now you'd be importing the entire file to `apiError`.